grails-app/services/org/pih/warehouse/receiving/ReceiptService.groovy

```diff
@@ -248,6 +249,19 @@ class ReceiptService {
             }
         }
 
+        // Send notification email on completed receiving
+        if (completed) {
```

Comment:
I don't mind that we're only sending when the whole shipment has been received, but I interpreted Kelsey's requirements as "send whenever any quantity was received". It seems like this would be a quick fix once Kelsey has UAT'd the ticket so let's leave it for now.

Reply:
completed means sending PARTIAL receiving. It can be either fully received or just partially. The same method is used just to save partial receipt.